superbit_lensing/run_test_pipe.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
Does this supersede the older `pipe_test.py` file? If so, the older file should be deleted.

Reply:
No. I realized that the `run_pipe.py` script was explicitly setup to only work for a pipe test, which was annoying. So I moved it to `run_test_pipe.py` (maybe that should have been `run_pipe_test.py`...) and have made `run_pipe.py` a more general script to run any pipe config.

Reply:
I just renamed it to `run_pipe_test.py` as I think that is clearer.